bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -72,6 +73,37 @@ type Tuple []TupleElement
 // an instance of this type.
 type UUID [16]byte
 
+// Versionstamp .
+type Versionstamp struct {
```

Comment:
I think the godocs on these new types good be more descriptive.

Reply:
👍